src/app/shared/models/course.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Time } from '@angular/common';
+
+export interface Course {
+    id: string;
+    title: string;
+    creationDate: Date;
+    duration: Time;
+    description: string;
+}
+
+export class CourseModel implements Course {
```

Comment:
feel free to remove this class and just use the interface for object creation =)

Reply:
@SteveRusin Classes were removed from model.ts files.